runtime/oti/j9nonbuilder.h

```diff
@@ -4877,6 +4882,18 @@ typedef struct J9InternalVMFunctions {
 #if JAVA_SPEC_VERSION >= 19
 	BOOLEAN (*createContinuation)(struct J9VMThread *currentThread, j9object_t continuationObject);
 #endif /* JAVA_SPEC_VERSION >= 19 */
+#if JAVA_SPEC_VERSION >= 16
+	void * ( *createUpcallThunk)(struct J9UpcallMetaData *data);
+	void * ( *getArgPointer)(struct J9UpcallNativeSignature *nativeSig, void *argListPtr, int argIdx);
+	void * ( *allocateUpcallThunkMemory)(struct J9UpcallMetaData *data);
+	void ( *doneUpcallThunkGeneration)(struct J9UpcallMetaData *data, void *thunkAddress);
+	void (JNICALL *icallVMprJavaUpcall0)(struct J9UpcallMetaData *data, void *argsListPointer);
```

Comment:
I strongly object to the `icallVMpr` naming here - those are JIT to interpreter linkage helpers which have nothing to do with C.

Reply:
Agreed and replaced `icallVMprJavaUpcall` with `native2InterpJavaUpcall` at multiple places.